src/Constructorio_NET.Tests/client/modules/QuizzesTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Constructorio_NET.Models;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+namespace Constructorio_NET.Tests
+{
+    [TestFixture]
+    public class QuizzesTest
```

Comment:
Can we add a test that checks if all the attributes in a quiz question are there, correct? Like the description, id, title, images...

That way we can be sure that the attributes deserialized correctly.